src/Model/Notification.php

```diff
@@ -23,7 +23,10 @@
 
 use Friendica\BaseModel;
 use Friendica\Content\Text\BBCode;
+use Friendica\Content\Text\Plaintext;
+use Friendica\Core\Logger;
 use Friendica\Database\Database;
+use Friendica\DI;
```

Comment:
We use a specific repository to create this class. Please add the needed classes at the constructor (injection) so we won't need the `DI::` calls at all :)

Reply:
This class is some kind of bastard since it contains both static and non static functions. The problem is that the class is occupying the name "notification" although is is ment as a model class for the "notify" table. Since we also have the the "notification" table which eventually will replace the "notify" table I would like to separate this class into a "notify" and a "notification" class. But since it belongs to a whole conglomerate of multiple classes, I'm totally unsure how this can be done.